src/btree.c

```diff
@@ -3157,6 +3156,109 @@ btree_count_in_range_by_iterator(cache        *cc,
    btree_iterator_deinit(&btree_itor);
 }
 
+/* Print offset table entries, 4 entries per line, w/ auto-indentation. */
+static void
+btree_print_offset_table(platform_log_handle *log_handle, btree_hdr *hdr)
+{
+   platform_log(log_handle, "-------------------\n");
+   platform_log(log_handle, "Offset Table num_entries=%d\n", hdr->num_entries);
+
+   uint64 nentries = btree_num_entries(hdr);
+   char   fmtstr[30];
+   snprintf(fmtstr,
+            sizeof(fmtstr),
+            "[%%%s] %%-8u",
+            ((nentries < 10)     ? "d"
+             : (nentries < 100)  ? "2d"
+             : (nentries < 1000) ? "3d"
+                                 : "4d"));
+
+   for (int i = 0; i < btree_num_entries(hdr); i++) {
+      // New-line every n-offset entries
+      if (i && ((i % 4) == 0)) {
+         platform_log(log_handle, "\n");
+      }
+      platform_log(log_handle, fmtstr, i, btree_get_table_entry(hdr, i));
+   }
+   platform_log(log_handle, "\n");
+}
+
+static void
+btree_print_index_node(platform_log_handle *log_handle,
+                       btree_config        *cfg,
+                       uint64               addr,
+                       btree_hdr           *hdr)
+{
+   data_config *dcfg = cfg->data_cfg;
+   platform_log(log_handle, "**  INDEX NODE \n");
+   platform_log(log_handle, "**  hdrptr: %p\n", hdr);
```

Comment:
I prefer printing `Header pointer:`

Reply:
Old code (before this work) was: `3180       platform_log(log_handle, "**  ptr: %p\n", hdr);`
Now Changed to: `3188    platform_log(log_handle, "**  Header ptr: %p\n", hdr);`